drivers/remote/api/api.go

```diff
@@ -30,6 +34,9 @@ type CreateNetworkRequest struct {
 
 	// A free form map->object interface for communication of options.
 	Options map[string]interface{}
+
+	// IPData contains the address pool information for this network
```

Comment:
Can you also remove `EndpointInterface` from this file ? and replace with anything that might be required for remote drivers ?